source/_docs/wordpress-export.md

```diff
@@ -77,26 +72,18 @@ mysqldump -uUSERNAME -pPASSWORD DATABASENAME > $TARGET/db.sql
 # Compress the backup.
 gzip $TARGET/db.sql
 ```
-### Export WordPress Files
+## Files
 Export a tar.gz or .zip file of your files directory, which was intentionally omitted from the codebase import. These files are not tracked in Git; instead, they are stored in Valhalla, our network file system.
+
 ```php
 TARGET=~/Desktop
 SOURCE=~/Projects/mysite
 cd $SOURCE/wp-content/uploads
 tar -czf $TARGET/files.tar.gz .
 ```
-## Prepare Your Archive for Import
-You now have three individual files that make one complete archive of your WordPress site. The import screen allows you to toggle between uploading your archive files or supplying a remote URL (e.g. Amazon S3, Dropbox, your existing server, etc.) to fetch the archives.
-
-You can upload each of your archive files separately, or package the archives within a single file. The max file upload import size is 100MB total. URL imports are limited to 500MB per input.
-
-
-<div class="alert alert-danger" role="alert">
-<h4 class="info">Warning</h4><p>Modify Dropbox URLs to end in <code>dl=1</code> instead of the default <code>dl=0</code>. This forces a download of your archive and avoids the Dropbox landing page.</p></div>
+## Next Steps
 
-Only provide one `.sql` file; if multiple are present the import will fail.
+Depending on your previous hosting platform, you can use tools like `rsync` or SFTP to copy these archive files locally. Follow either of the guides below to manually migrate your site.
```

Comment:
@rachelwhitton that's covered by the manual migration guide. I think that this comment and the one on L10 can be solved by some better crosslinking, both on this guide and the manual migration one.